jarviscli/plugins/chess.py

```diff
@@ -0,0 +1,28 @@
+# There needs to be a binary named 'stockfish' in the PATH for this to work.
+
+import os
+from plugin import plugin
+from stockfish import Stockfish
+
+
+@plugin('chess')
+class Chess():
+    def __call__(self, jarvis, s):
+        fish = Stockfish()
+        while (True):
+            jarvis.say("Choose your option:")
+            jarvis.say("1. Moves")
+            jarvis.say("2. FEN")
+            option = jarvis.input("Choice: ")
+            if (option == '1'):
+                moves = jarvis.input("Moves: ")
+                print(moves.split())
+                fish.set_position(moves.split())
+                jarvis.say(fish.get_best_move())
+            elif (option == '2'):
+                fen = jarvis.input("FEN: ")
+                if (fish.is_fen_valid(fen)):
+                    fish.set_fen_position(fen)
+                    jarvis.say(fish.get_best_move())
```

Comment:
You should catch ValueError at this line

Moves: a2
['a2']
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/data/git/Jarvis/jarviscli/CmdInterpreter.py", line 286, in try_do
    do(self, s)
  File "/data/git/Jarvis/jarviscli/plugin.py", line 208, in run
    self._backend[0](jarvis.get_api(), s)
  File "/data/git/Jarvis/jarviscli/plugins/chess.py", line 20, in __call__
    fish.set_position(moves.split())
  File "/home/phil/git/Jarvis/env/lib/python3.8/site-packages/stockfish/models.py", line 222, in set_position
    self.make_moves_from_current_position(moves)
  File "/home/phil/git/Jarvis/env/lib/python3.8/site-packages/stockfish/models.py", line 238, in make_moves_from_current_position
    raise ValueError(f"Cannot make move: {move}")
ValueError: Cannot make move: a2